app/ledger.js

```diff
@@ -111,6 +111,18 @@ const doAction = (action) => {
       if (action.key === settings.PAYMENTS_CONTRIBUTION_AMOUNT) return setPaymentInfo(action.value)
       break
 
+    case appConstants.APP_CHANGE_SITE_SETTING:
+      if (action.key === 'ledgerPaymentsShown') {
+        // TODO
+      }
+      break
+
+    case appConstants.APP_REMOVE_SITE_SETTING:
+      if (action.key === 'ledgerPaymentsShown') {
+        // TODO
```

Comment:
@mrose17 this event indicates that a deleted ledger synopsis entry has been un-deleted so it should be included again